com.github.pyfa_org.Pyfa.json

```diff
@@ -0,0 +1,347 @@
+  "finish-args" : [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--filesystem=xdg-config/gtk-3.0"
+  ],
+  "modules" : [
+    {
+      "name" : "python-3.7",
+      "config-opts" : [
+        "--prefix=${FLATPAK_DEST}",
+        "--enable-shared",
+        "--with-computed-gotos",
+        "--enable-ipv6",
+        "--with-system-expat",
+        "--with-dbmliborder=gdbm:ndbm",
+        "--enable-loadable-sqlite-extensions",
+        "--with-ensurepip",
+        "--with-setuptools=no"
+      ],
+      "sources" : [
+        {
+          "url" : "https://www.python.org/ftp/python/3.7.15/Python-3.7.15.tar.xz",
+          "sha256" : "5911475a07ac2b53d746e88a0716af6d2b4734941919136ea0d33fb9c75b9714",
+          "type" : "archive"
+        }
+      ],
+      "cleanup" : [
+        "/bin/2to3*",
+        "/bin/easy_install*",
+        "/bin/idle*",
+        "/bin/pydoc*",
+        "/bin/python*-config",
+        "/bin/pyvenv*",
+        "/include",
+        "/lib/debug",
+        "/lib/pkgconfig",
+        "/lib/python*/config",
+        "/lib/python*/config-3.7m*",
+        "/share",
+        "/lib/python*/pydoc_data",
+        "/lib/python*/test",
+        "/lib/python*/*/test",
+        "/lib/python*/*/tests",
+        "/lib/python*/lib-tk/test",
+        "/lib/python*/lib-dynload/_*_test.*.so",
+        "/lib/python*/lib-dynload/_test*.*.so",
+        "/lib/python*/idlelib",
+        "/lib/python*/tkinter*",
+        "/lib/python*/turtle*",
+        "/lib/python*/lib2to3*",
+        "/lib/python2.7/config/libpython2.7.a"
+      ],
+      "post-install" : [
+        "chmod 644 $FLATPAK_DEST/lib/libpython3.7m.so.1.0",
+        "chmod 644 $FLATPAK_DEST/lib/libpython3.so"
+      ]
+    },
+    {
+      "//": "Taken from shared-modules/gtk2/gtk2.json with a few more patches.",
+      "name": "gnome-themes-extra",
+      "rm-configure": true,
+      "config-opts": [
+          "--disable-dependency-tracking",
+          "--disable-gtk3-engine"
+      ],
+      "cleanup": [
+          "/share/themes/Adwaita/gtk-3.0",
+          "/share/themes/Adwaita-dark/gtk-3.0",
+          "/share/themes/HighContrast/gtk-3.0",
+          "*.la"
+      ],
+      "sources": [
+          {
+              "type": "archive",
+              "url": "https://download.gnome.org/sources/gnome-themes-extra/3.28/gnome-themes-extra-3.28.tar.xz",
+              "sha256": "7c4ba0bff001f06d8983cfc105adaac42df1d1267a2591798a780bac557a5819"
+          },
+          {
+              "type": "shell",
+              "//": "We want to avoid generating icons as its 99% of the build time and gnome runtime has it",
+              "commands": [
+                  "sed -i 's/icons//' themes/HighContrast/Makefile.am"
+              ]
+          },
+          {
+              "type": "script",
+              "commands": [
+                  "autoreconf -fsi"
+              ]
+          }
+      ],
+      "modules": [
+          {
+              "name": "gtk2",
```

Comment:
please use the one from shared-modules

https://github.com/flathub/shared-modules/

Reply:
Done